src/ui/dialogs/settings.ts

```diff
@@ -37,15 +37,38 @@ class StartPage extends Page {
 class ClientSection extends Section {
    protected generateContentElement(): JQuery {
       let contentElement = new List();
+      contentElement.append(new ListItem(
+         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
+         undefined,
+         undefined,
+         undefined
+      ));
 
       contentElement.append(new ListItem(
          Translation.t('Language'),
-         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
          undefined,
          undefined,
          this.getLanguageSelectionElement()
       ));
 
+      contentElement.append(new ListItem(
+         Translation.t('trusted_domains'),
```

Comment:
I think for the user it's not obvious what a trusted domain is. We need an explanation.

Reply:
added further infos